command/ibft/ibft_propose.go

```diff
@@ -0,0 +1,139 @@
+package ibft
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-sdk/command/helper"
+	"github.com/0xPolygon/polygon-sdk/consensus/ibft/proto"
+	ibftOp "github.com/0xPolygon/polygon-sdk/consensus/ibft/proto"
+	"github.com/0xPolygon/polygon-sdk/types"
+)
+
+// IbftPropose is the command to query the snapshot
+type IbftPropose struct {
+	helper.Meta
+}
+
+// DefineFlags defines the command flags
+func (p *IbftPropose) DefineFlags() {
+	if p.FlagMap == nil {
+		// Flag map not initialized
+		p.FlagMap = make(map[string]helper.FlagDescriptor)
+	}
+
+	p.FlagMap["addr"] = helper.FlagDescriptor{
+		Description: "Address of the account to be voted for",
+		Arguments: []string{
+			"ETH_ADDRESS",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      false,
+	}
+
+	p.FlagMap["vote"] = helper.FlagDescriptor{
+		Description: fmt.Sprintf(
+			"Proposes a change to the validator set. Possible values: [%s, %s]", positive, negative,
+		),
+		Arguments: []string{
+			"VOTE",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      false,
+	}
+}
+
+// GetHelperText returns a simple description of the command
+func (p *IbftPropose) GetHelperText() string {
+	return "Proposes a new candidate to be added or removed from the validator set"
+}
+
+func (p *IbftPropose) GetBaseCommand() string {
+	return "ibft propose"
+}
+
+// Help implements the cli.IbftPropose interface
```

Comment:
cli.IbftPropose interface, same as in the ibft_candidates.go

Reply:
I'm not sure what you meant by this?

Reply:
me neither now :)